docs/doxygen/config.doxyfile

```diff
@@ -270,6 +270,9 @@ TAB_SIZE               = 4
 # a double escape (\\{ and \\})
 
 ALIASES                =
+# Aliases for tables on the Style Guide page.
+ALIASES += formattable{1}="This table contains the formats for the names of the most common SDK \1. It is not intended to be comprehensive. <b>Bold</b> text indicates a variable but required part of the \1 name. <i>Italic</i> text indicates a variable but optional part of the \1 name.^^^^| Format | Applies to | Example |^^| ------ | ---------- | ------- |"
```

Comment:
Change `\1` to `\1s` and `\1's`. I.e. it should be "file's name" and not "files name"

Reply:
Looks like this adds a trailing s but I kept the \1' where needed